includes/derivatives.inc

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * @file
+ * Fixes Derivative Policy issues.
```

Comment:
File doc comments should be a bit more general - like `Contains derivative functionality for WWU` or something. Otherwise if we add to this file in the future (say, a custom thumbnail derivative for this client), the comment would no longer reflect what's in the file